win32ss/user/ntuser/winpos.c

```diff
@@ -2048,7 +2048,7 @@ co_WinPosSetWindowPos(
        * change.
        */
       if ( ( VisBefore != NULL &&
```

Comment:
Two comments:
1. The second opening parenthesis (and the previous-to-last one at line 2054) are redundant and can be removed;

Reply:
Removed extra parenthesis. Thanks.